prow/cmd/line/main.go

```diff
@@ -334,12 +334,16 @@ func (c *testClient) TestPRJenkins() error {
 		c.tryCreateStatus("", github.StatusError, "Error starting build.", testInfra)
 		return err
 	}
-	eq, err := c.JenkinsClient.Enqueued(b)
-	if err != nil {
-		c.tryCreateStatus("", github.StatusError, "Error queueing build.", testInfra)
-		return err
+	eq := false
+	for !eq { // Wait for it to appear in the queue
+		time.Sleep(10 * time.Second)
```

Comment:
This doesn't actually delay the build at all, it just takes a little longer to get through the queue.